### ISSUE TITLE
List of Topics and Setting Bed Temp

### ISSUE THREAD

**Issue by BlueWings172** [jneilliii/OctoPrint-MQTTSubscribe#35]
Hello

Thanks for the great plugin. I just need help with the below:

1- From where I can get the list of topics that I can publish to ?

2- I'm trying to set the bed temp through MQTT so I entered the below in the editor but it not working. It does work for static temperature only:

![image](https://user-images.githubusercontent.com/65966061/168796159-1dc81fed-efe4-4269-90d8-d43a5fb6f802.png)

Thanks

**Comment by jneilliii** [comment]
I'll have to experiment for '#2', but '#1' is any topic you want to use, that's just how mqtt works.

**Comment by BlueWings172** [comment]
Thanks for the answer. I'm new to MQTT and learning. 

I figured out the solution to the 2nd issue. I should've removed the double quotes surrounding the variable and when I changed the line to:  `{"command": "target","target": {0}}`  , it worked. Im posing the correct config below incase someone else is trying to do the same thing:

![image](https://user-images.githubusercontent.com/65966061/168807875-f24f8e05-79e8-4968-a61e-54d524706ce7.png)
